code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -92,7 +96,9 @@
 	var/real_spines = C.dna.features["spines"]
 
 	. = ..()
-
+	C.mob_surgery_speed_mod -= 0.35
```

Comment:
Have you actually tested this in game? Because this is an **enormous** penalty. This is comparable to the difference between doing surgery on the floor, and doing it on a surgical table with a linked operating computer. This is larger than the theoretical maximum penalty of impure cryostylane, and will be the largest surgical speed penalty in the codebase.

The main criticism of the medical/surgery system is that it's too slow.

Reply:
what the fuck is this shit, why

Reply:
i checked ingame it didnt feel that bad, ill see during the testmerge though

Reply:
You should also be aware of how cryostylane is implemented:
https://github.com/tgstation/tgstation/blob/9e72e7af249c104ca5d69e1cc12efad3fc7a3e71/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm#L242-L245

Administering and purging 1u of cryostylane will set mob_surgery_speed_mod to 1.

Reply:
i already factored this in

Reply:
alright its nerfed